content/en/synthetics/dashboards/_index.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Synthetics Dashboards
+kind: documentation
+description: Use the out-of-the-box Synthetics API Tests dashboard to proactively monitor your entire stack.  
+further_reading:
+- link: '/synthetics/'
+  tag: 'Documentation'
+  text: 'Learn about Synthetic Monitoring'
+- link: '/synthetics/ci_results_explorer'
+  tag: 'Documentation'
+  text: 'Learn about the CI Results Explorer'
+---
+
+## Overview
+
+When you create a Synthetics test, Datadog [collects data][1] and generates dashboards about your stack, browser applications, or overall tests' performance, private locations, and events. 
+
+Access your Synthetics dashboards by filtering for `Synthetics` in the search query of the [**Dashboard List**][2] or by clicking on the dropdown menu under [**Dashboards**][3].
+
+{{< img src="synthetics/dashboards/synthetic_tests_dashboards.png" alt="Synthetic Monitoring Dashboards" style="width:100%;">}}
+
+{{< whatsnext desc="You can explore the following out-of-the-box Synthetics dashboards:" >}}
+  {{< nextlink href="/synthetics/dashboards/api_test" >}}<u>API Test Performance</u>: Monitor your entire stack. {{< /nextlink >}}
+  {{< nextlink href="/synthetics/dashboards/browser_test" >}}<u>Browser Test Performance</u>: Create browser tests to monitor and test your applications and services. {{< /nextlink >}}
+  {{< nextlink href="/synthetics/dashboards/test_summary" >}}<u>Test Summary</u>: See insights about your Synthetic tests by region, environment, or team. {{< /nextlink >}}
+{{< /whatsnext >}}
+
+## Interact with Synthetics dashboards
+
+You can customize your [dashboards][4] or directly explore the CI jobs executing your Synthetics tests in the [CI Results Explorer][5].
```

Comment:
```suggestion
You can customize your [dashboards][4] and clone them by team, environment or region utilising template variables.
```
Not sure how this sounds, I don't think we should mention CI at this point, it would only be relevant for the Test Summary dashboard